pages/_app.tsx

```diff
@@ -10,16 +10,45 @@ import type { AppProps } from 'next/app';
 import { ChakraProvider } from '@chakra-ui/react';
 import theme from '../theme';
 import '../styles/Mobile.css';
+import '../styles/globals.css';
 import Head from 'next/head';
 import { Web3Provider } from '../contexts/Web3Context';
 import Layout from '../components/Layout';
 
 function MyApp({ Component, pageProps }: AppProps) {
+
+  // Production MixPanel -> Bankless DAO
+  const MIX_PANEL_ID = '0715c4ae0cba2709f3d752f073ef2824'
```

Comment:
please if you would confirm this is not sensitive data that should be placed in a .env file / enviroment config?

Reply:
@ken333135 Can you confirm we are good here? I'm sure it's not that big of a deal. It amounts to a google analytics tag.

Reply:
Is there a difference between a sandbox and production account id? It looks like it's not super sensitive based on the code below, since it can be seen on the client side.

Reply:
I missed that the development ID is located below in a comment. I agree with @geositta that it might be helpful to have an environment config for this data in the future.

Reply:
yea the id is just like a google analytics tag. should not be an issue.
will take note to move this to a environment config file in the future!